Tasks/month.js

```diff
@@ -15,12 +17,14 @@ Months = [
   'dec',
 ];
 
-Month = (s) => {
-  l = Months.length;
-  for (let i = 0; i < l; i++) {
-    if (s.toLowerCase().startsWith(Months[i])) return i + 1;
+const getNumOfMonth = (month) => {
+  const lowerCase = month.toLowerCase();
+  for (let i = 0; i < arrayOfMonths.length; i++) {
```

Comment:
Here you can change the variable 'i' to 'index' for example. It isn't necessary, but the code will look more understandable